testing/sdk_networks.py

```diff
@@ -1,10 +1,18 @@
+import json
+
 import shakedown
 
 
 def check_task_network(task_name, expected_network_name="dcos"):
     """Tests whether a task (and it's parent pod) is on a given network
     """
     _task = shakedown.get_task(task_id=task_name, completed=False)
+
+    if type(_task) == list:
```

Comment:
type testing is usually wrong.  does get_task actually return an object OR a list?

From the code:

```pythondef get_tasks(task_id='', completed=True):
    """ Get a list of tasks, optionally filtered by task id.

        :param task_id: task ID
        :type task_id: str
        :param completed: include completed tasks?
        :type completed: bool

        :return: a tuple of tasks
        :rtype: tuple
    """
```
If this is right, your code will never get a list.  

Probably your code should be 

tasks = shakedown.get_tasks(...)
assert len(tasks) == 1, ...